footprint: [[-0.41,0.345],[-0.41,-0.345],[0.41,-0.345],[0.41,0.345]]
footprint_padding: 0.04
map_type: costmap
